presto-main/pom.xml

```diff
@@ -36,6 +36,11 @@
             <artifactId>commons-math3</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Is there any way to do this without adding a dependency on commons?

Reply:
The security scanner recommends using a proper library for encoding output (such as the OWASP ESAPI Encoding module, OWASP Java HTML Sanitizer, OWASP AntiSamy, or Apache Shiro). In the IBM Pesto main module, the org.apache.commons.lang3 dependency is already included, so there is no need to add any additional dependencies for this fix. I also tried to apply the same changes for OSS, as this dependency is already used by other modules.

If I address the question, the answer is yes, we can manually validate special characters. While manual escaping works, it can become difficult when handling more complex HTML/JS edge cases. Libraries like StringEscapeUtils are more robust and thoroughly tested.